client/src/features/admin/AddSessionEnvironmentButton.tsx

```diff
@@ -86,13 +86,13 @@ function AddSessionEnvironmentModal({
         addSessionEnvironment({
           container_image: data.container_image,
           name: data.name,
-          default_url: data.default_url.trim() ? data.default_url : undefined,
-          description: data.description.trim() ? data.description : undefined,
+          default_url: data.default_url?.trim() ? data.default_url : undefined,
```

Comment:
For here and everywhere else: should we instead have this?

```suggestion
          default_url: data.default_url?.trim() || undefined,
```

Otherwise we send `untrimmed` strings to the API.

This can be added to a new issue to do later.